app/assets/stylesheets/_screen.scss

```diff
@@ -361,6 +365,61 @@ a.button {
   margin: 0;
 }
 
+.StoryLink {
```

Comment:
I think we're standardizing class names to use dash-notation, like `story-link`.